packages/core/src/meta/MetaManagerImpl.ts

```diff
@@ -1,13 +1,18 @@
 import MetaManager from './MetaManager';
 
+export interface MetaTag {
```

Comment:
i know that this probably isn't the right place for this (and i should've used `type`), so where to put it?